src/canvas/ElementsManager/CanvasElements.ts

```diff
@@ -0,0 +1,119 @@
+import { getEnv, getFabricDocument } from '../../env';
+import type { TSize } from '../../typedefs';
+import {
+  createCanvasElement,
+  makeElementUnselectable,
+  setStyle,
+} from '../../util';
+import type { CSSDimensions } from '../../util/dom_misc';
+import { allowTouchScrolling } from '../../util/dom_misc';
+import { setCSSDimensions } from '../../util/dom_misc';
+import type { CanvasItem } from './StaticCanvasElements';
+import { StaticCanvasElements } from './StaticCanvasElements';
+import { setCanvasDimensions } from './util';
+
+export class CanvasElements extends StaticCanvasElements {
```

Comment:
CanvasElement would be like a CanvasDOMManager?

Reply:
Yes
A better name indeed